src/executors.jl

```diff
@@ -365,13 +356,56 @@ function dispatch!(exec::Executor, graph::DispatchGraph; throw_error=true)
 
     len = length(graph.nodes)
     info(logger, "Executing $len graph nodes.")
-    res = asyncmap(f, 1:len; ntasks=div(len * 3, 2))
+
+    asyncmap(f1, 1:len; ntasks=div(len * 3, 2))
```

Comment:
Does this need to be an `asyncmap`? What would be the effect of making this just a `foreach`, in typical conditions and error conditions?